README.md

```diff
@@ -105,6 +105,16 @@ To customize Radicale configuration, either:
 
 Then puts these two files in a directory and use the config volume `-v /my_custom_config_directory:/config` when running the container.
 
+## Contributing
+
+First install the test dependencies
+
+`pip install --user -r requirements.txt`
```

Comment:
As the requirements file is only for testing, what do you think of naming it `requirements-test.txt` (eg. https://github.com/uber/tchannel-python/blob/master/requirements-test.txt ) ?